### PR TITLE
adding insight report to the report documentation page

### PR DESCRIPTION
**Pull request by obahidika**
adding the insight report feature documentation

### CODE REVIEW

File: Teams/rooms/health-and-usage-reports.md [hunk 1]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
 <!--!![A screenshot of room audio quality](../media/health-and-usage-016.png)-->
 
 To view stream details for the room, click the Session Start Time.
+
+
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:171]:
```suggestion
```

File: Teams/rooms/health-and-usage-reports.md [hunk 2]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
...
+## Insight Report
+
+The insight report is located under the report section of the pro management portal and provides a detailed overview of all the activities completed by our services during a specific time range, including the types of actions performed such as detections, remediation, and update. This report highlights the value we provide through actions to resolve rooms’ issue and how much time and money we help you save by doing that. 
+To view insight report of all rooms, select Report > Insight Report (Preview)
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:175]:
```suggestion
## Insight report

The insight report is located under the report section of the pro-management portal. It provides a detailed overview of all the activities completed by our services during a specific time range, including the types of actions performed, such as detections, remediation, and update. This report highlights the value we provide through actions to resolve rooms issues and how much time and money we help you save by doing that. 
To view the insight report of all rooms, select Report > Insight Report (Preview).
```

File: Teams/rooms/health-and-usage-reports.md [hunk 3]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
...
+
+
+## Insight Report
+
+The insight report is located under the report section of the pro management portal and provides a detailed overview of all the activities completed by our services during a specific time range, including the types of actions performed such as detections, remediation, and update. This report highlights the value we provide through actions to resolve rooms’ issue and how much time and money we help you save by doing that. 
+To view insight report of all rooms, select Report > Insight Report (Preview) 
+
+## Actions: 
+ The actions section of the report shows a timeline view and aggregate count of all actions completed each day by the service. Action types include Detection, Remediation, and Updates. Details on the actions can be viewed by selecting "See details”. Once click on the “see details”, a new panel will come with two sections:  Tickets and Updates. The ticket shows a list of tickets with corresponding hours saved. Select the ticket to show an expanded view of each action taken on the respective ticket and hour saved. the Updates section shows a list of each update applied and corresponding tally of devices that successfully completed the update.
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:178]:
```suggestion
## Actions

 The Actions section of the report shows a timeline view and aggregate count of all actions completed each day by the service. Action types include detection, remediation, and updates. Details on the actions can be viewed by selecting **See details**. Once you have selected **See details**, a new panel will appear with two sections; Tickets and Updates. The Tickets section shows a list of tickets with corresponding hours saved. Select the Tickets section to show an expanded view of each action taken on the respective ticket and the hours saved. the Updates section shows a list of each update applied and the corresponding tally of devices that successfully completed the update.  
```

File: Teams/rooms/health-and-usage-reports.md [hunk 4]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
...
+To view insight report of all rooms, select Report > Insight Report (Preview) 
+
+## Actions: 
+ The actions section of the report shows a timeline view and aggregate count of all actions completed each day by the service. Action types include Detection, Remediation, and Updates. Details on the actions can be viewed by selecting "See details”. Once click on the “see details”, a new panel will come with two sections:  Tickets and Updates. The ticket shows a list of tickets with corresponding hours saved. Select the ticket to show an expanded view of each action taken on the respective ticket and hour saved. the Updates section shows a list of each update applied and corresponding tally of devices that successfully completed the update.  
+ 
+## Detection, investigation, and remediation: 
+The “Detection, investigation, and remediation” section displays two counts, respective of the specified time range: Meeting issues prevented and Incidents remediated.
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:181]:
```suggestion
## Detection, investigation, and remediation

The Detection, investigation, and remediation section displays two counts respective of the specified time range: Meeting issues prevented and Incidents remediated.
```

File: Teams/rooms/health-and-usage-reports.md [hunk 5]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
...
+
+## Updates:
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:183]:
```suggestion
## Updates

```

File: Teams/rooms/health-and-usage-reports.md [hunk 6]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
...
+To view insight report of all rooms, select Report > Insight Report (Preview) 
+
+## Actions: 
+ The actions section of the report shows a timeline view and aggregate count of all actions completed each day by the service. Action types include Detection, Remediation, and Updates. Details on the actions can be viewed by selecting "See details”. Once click on the “see details”, a new panel will come with two sections:  Tickets and Updates. The ticket shows a list of tickets with corresponding hours saved. Select the ticket to show an expanded view of each action taken on the respective ticket and hour saved. the Updates section shows a list of each update applied and corresponding tally of devices that successfully completed the update.  
+ 
+## Detection, investigation, and remediation: 
+The “Detection, investigation, and remediation” section displays two counts, respective of the specified time range: Meeting issues prevented and Incidents remediated.
+
+## Updates: 
+The Updates section provides a count of total updates either scheduled, in progress, or completed across the devices enrolled in the service.
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:184]:
```suggestion
## Updates
 
The Updates section provides a count of total updates either scheduled, in progress or completed across the devices enrolled in the service.
```

File: Teams/rooms/health-and-usage-reports.md [hunk 7]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
...
+
+## Estimated savings 
+The estimated hours saved from the actions completed are converted to a monetary amount based on a default rate of $50 USD/hour for IT professionals. To customize, select See details in this section.  
+The detail pane allows you to input a custom hourly rate and corresponding currency to get a better monetary estimate.
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:188]:
```suggestion
## Estimated savings 
The estimated hours saved from the actions completed are converted to a monetary amount based on a default rate of $50 USD/hour for IT professionals. To customize the rate, select _See details_ in this section.
The Detail pane allows you to input a custom hourly rate and corresponding currency to get a better monetary estimate.  
```

File: Teams/rooms/health-and-usage-reports.md [hunk 8]
```diff
@@ -167,3 +167,28 @@ The **Calls** table shows meetings in which the room participated in a Teams cal
...
+The Updates section provides a count of total updates either scheduled, in progress, or completed across the devices enrolled in the service.
+
+## Estimated savings 
+The estimated hours saved from the actions completed are converted to a monetary amount based on a default rate of $50 USD/hour for IT professionals. To customize, select See details in this section.  
+The detail pane allows you to input a custom hourly rate and corresponding currency to get a better monetary estimate.  
+
+## User experience 
+This section provides the average call rating by end users on a scale of 1-5 (5 being the best). 
+
+
```
Comment (JohanFreelancer9) [inline Teams/rooms/health-and-usage-reports.md:193]:
```suggestion
## User experience 
This section provides the average call rating by end-users on a scale of 1 to 5 (5 being the best).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @obahidika, kindly commit the suggested changes. Many thanks.

**Review by obahidika — COMMENTED** [review]
thanks for the suggestions

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
@obahidika 
Hi,

This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main). 

We encourage you to use the private repo for future contributions to Microsoft Learn.

Thanks!

**Review by altsou — APPROVED** [review]
Looks good to me.

**Pull request merged**